fpsim/locations/data_processing/breastfeeding_stats.R

```diff
@@ -0,0 +1,20 @@
+### Data to process DHS data for breastfeeding duration
+
+dir <- "directory" ## directory with data
+filename <- "filename" ## must be a KR (child recode) file from the DHS for automatic processing
```

Comment:
I think most other things come from the IR survey, so it might be worth flagging specifically that this would be the one piece of data coming from the KR survey

Reply:
That's definitely a good idea, since I had to go digging in the IR first!